src/compose/ComposeMenu.js

```diff
@@ -84,90 +77,128 @@ export const chooseUploadImageFilename = (uri: string, fileName: string): string
   return nameWithoutPrefix;
 };
 
-class ComposeMenuInner extends PureComponent<Props> {
-  static contextType = TranslationContext;
-  context: GetText;
+export default function ComposeMenuInner(props: Props): Node {
```

Comment:
nit: There's no longer a distinct "inner" vs "outer" component here, right? So this function should be called just `ComposeMenu`.

(Another angle on the same thing: it's the default export, so should have the same name as the file.)